app/scripts/services/tile-proxy.js

```diff
@@ -517,7 +540,8 @@ export const tileDataToPixData = (
   zeroValueColor,
   selectedRows,
   selectedRowsAggregationMode,
-  selectedRowsAggregationWithRelativeHeight
+  selectedRowsAggregationWithRelativeHeight,
```

Comment:
The parameters for this function are getting bulky. Just planting the seed of the thought of refactoring it at some point.